security/clamav/src/opnsense/service/conf/actions.d/actions_clamav.conf

```diff
@@ -35,7 +35,7 @@ type:script_output
 message:Check or install signatures
 
 [version]
-command:/usr/local/bin/clamconf | grep "Version:\|main.cvd: version\|.cld: version\|Total number of signatures:"
+command:/usr/local/bin/clamconf | grep "Version:\|.cvd: version\|.cld: version\|Total number of signatures:"
```

Comment:
got it even better:
```sh
(echo -n "{" ; clamconf | sed -n '/Database information/,/^$/p' | tail +2 | grep : | while read line ; do key=`echo  $line | cut -d':' -f1`; value=`echo  $line | cut -d':' -f2` ; echo "\"$key\": \"$value\""; done | tr "\n", "," | sed 's/,$//' ;  echo "}" )
```

Reply:
this is too long and fragile to maintain inside the action.conf file anymore